tests/PySys/Readme.md

```diff
@@ -0,0 +1,109 @@
+## Overview System Tests
+
+System tests for thin-edge in this folder are written in Python with
+the PySys system test framework.
+
+See also:
+
+* [https://pysys-test.github.io/pysys-test/](https://pysys-test.github.io/pysys-test/)
+* [https://github.com/pysys-test/](https://github.com/pysys-test/)
+
+
+#### Running System tests manually on your device / Linux PC
+
+The system tests can be executed manually. In advance, you need a tenant in
+Cumulocity as well as a previously generated device certificate. The device
+needs to be able to connect to Cumulocity successfully.
+
+The test configuration is located in environment variables, which need to be
+set in in advance. Also compile the sawtooth_publisher, as it is used by some
+tests.
+
+    cargo build --example sawtooth_publisher
+
+These environment variables need to be exported in your shell:
+
+    export TEBASEDIR=~/thin-edge.io/
+    export EXAMPLEDIR=$HOME/thin-edge.io/target/debug/examples
+    export C8YUSERNAME= <your tenant>
+    export C8YPASS= <your password>
+    export C8YDEVICE= <dev name>
+    export C8YTENANT= <tenant id>
```

Comment:
When I connect to the c8y manually the T8YTENANT is not used at all. I did not get in which situation this is needed.